monadic-games-club.cabal

```diff
@@ -0,0 +1,36 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.36.1.
+--
+-- see: https://github.com/sol/hpack
+
+name:           monadic-games-club
+version:        0.1.0.0
+homepage:       https://github.com/silky/monadic-games-club
+license:        Hippocratic
```

Comment:
should actually be `OtherLicense` since `Hippocratic` is not recognised by Cabal, as far as I remeber

Reply:
unsurprising 😅